src/wast-parser.cc

```diff
@@ -1499,28 +1499,85 @@ Result WastParser::ParsePlainInstr(std::unique_ptr<Expr>* out_expr) {
   return Result::Ok;
 }
 
+// Current Simd const type is V128 const only.
+// The current expected V128 const lists is:
+// i32 0xXXXXXXXX 0xXXXXXXXX 0xXXXXXXXX 0xXXXXXXXX
+Result WastParser::ParseSimdConst(Const* const_, Type in_type, int32_t nSimdConstBytes) {
+  WABT_TRACE(ParseSimdConst);
+
+  // Parse the Simd Consts according to input data type.
+  switch (in_type) {
+    case Type::I32: {
+      const_->loc = GetLocation();
+      int nCount = nSimdConstBytes/sizeof(uint32_t);
```

Comment:
We can hardcode this for now, since we know that `nSimdConstBytes` is always `sizeof(v128)`.

Reply:
The reason why I didn't use hardcode here is: 
I want to this function can be used for future 256 or 512 simd type.